protocol/devops/README.md

```diff
@@ -0,0 +1,75 @@
+# DevOps Protocol
+
+A set of guidelines, checklists and tutorials for deploying web apps in a sane
+and reproduceable way.
+
+## Table of Contents
+
+* [Continuous Integration](#continuous-integration)
+* [Servers](#server-setup)
+* [Static websites](#static-websites)
+* [DNS](#server-setup)
+* [SSL](#server-setup)
+    * [Sabayon Samples](#sabayon-samples)
+
+## Continuous Integration
+
+* Use [SemaphoreCI] ([TravisCI] is also fine for open source, but tends to be
+    slower)
+* Set up automatic deploys to staging (from the `master` branch), and manual
+    deploys to production
+* Ensure the CI server is capable of deploying to all environments, so that no
```

Comment:
Ok. I just want to add that you can also easily do that through the heroku dashboard, the CLI is just a nice to have. Anyone can promote a staging application to promotion with just a click.